test/src/test/java/org/corfudb/runtime/collections/PersistedCorfuTableTest.java

```diff
@@ -29,15 +29,17 @@
 import org.corfudb.runtime.CorfuRuntime;
 import org.corfudb.runtime.CorfuRuntime.CorfuRuntimeParameters;
 import org.corfudb.runtime.CorfuStoreMetadata;
+import org.corfudb.runtime.ExampleSchemas;
+import org.corfudb.runtime.ExampleSchemas.Person;
```

Comment:
Just a suggestion: the existing example schema `Adult` with age and child's age might be a more representative example of secondary indexes having both alias and non-alias than `Person`